src/runtime/app.js

```diff
@@ -6,20 +6,28 @@ module.exports = function () {
   const { PORT, CORS_SETTINGS } = require('../lib/constants')
 
   const app = express()
+  const logger = require('../lib/logger/index')
   const cors = require('cors')
   const { startPollers } = require('./pollers')
   const initDb = require('../lib/db')
+  const sentry = logger()
```

Comment:
this creates a new instance in each file. is it better to create only one instance inside `'../lib/logger/index'`?